js/services.js

```diff
@@ -0,0 +1,166 @@
+/* global ServiceConfig, Promise, Messenger */
+
+'use strict';
+
+(function(exports) {
+  /**
+   * This module is responsible to create and run services.
+   */
+  var Services = function() {};
+  Services.prototype = {
+
+    _dispatchReady: function() {
```

Comment:
it's really hard to name it, How about with full event name `_dispatchSystemMessageListenerReady`